index.bs

```diff
@@ -3225,22 +3263,28 @@ transport protocols are referred to as <dfn>roaming authenticators</dfn>.
     device=]. A [=public key credential=] [=bound credential|bound=] to a [=platform authenticator=] is called a <dfn>platform credential</dfn>.
 
 - A [=roaming authenticator=] is attached using cross-platform transports, called <dfn>cross-platform attachment</dfn>. Authenticators of this class are removable from, and
-    can "roam" among, [=client devices=]. A [=public key credential=] [=bound credential|bound=] to a [=roaming authenticator=] is called a <dfn>roaming
+    can "roam" between, [=client devices=]. A [=public key credential=] [=bound credential|bound=] to a [=roaming authenticator=] is called a <dfn>roaming
     credential</dfn>.
 
 Some [=platform authenticators=] could possibly also act as [=roaming authenticators=] depending on context. For example, a
 [=platform authenticator=] integrated into a mobile device could make itself available as a [=roaming authenticator=] via
-Bluetooth. In this case the [=client=] would recognize it only as a [=roaming authenticator=], and not as a [=platform
-authenticator=].
-
-A primary use case for [=platform authenticators=] is to register a particular [=client device=] as a "trusted device" available
-as a [=something you have=] [=authentication factor=] for future [=authentication=]. This gives the user the convenience benefit
+Bluetooth.
```

Comment:
s/ Bluetooth / Bluetooth, for example /   ...?

Reply:
The sentence begins with "For example", do we need one here as well?